system-int-test/src/cmr/system_int_test/utils/variable_util.clj

```diff
@@ -1,44 +1,37 @@
 (ns cmr.system-int-test.utils.variable-util
-  "This contains utilities for testing variable"
+  "This contains utilities for testing variables."
   (:require
    [clojure.string :as string]
-   [cmr.acl.core :as acl]
    [clojure.test :refer [is]]
    [cmr.common.mime-types :as mt]
    [cmr.common.util :as util]
-   [cmr.mock-echo.client.echo-util :as e]
+   [cmr.mock-echo.client.echo-util :as echo-util]
    [cmr.system-int-test.data2.core :as d]
    [cmr.system-int-test.system :as s]
    [cmr.system-int-test.utils.index-util :as index]
+   [cmr.system-int-test.utils.ingest-util :as ingest-util]
    [cmr.system-int-test.utils.metadata-db-util :as mdb]
    [cmr.system-int-test.utils.search-util :as search]
    [cmr.transmit.echo.tokens :as tokens]
    [cmr.transmit.variable :as transmit-variable]))
 
-(defn- get-system-ingest-update-acls
-  "Get a token's system ingest management update ACLs."
-  [token]
-  (-> (s/context)
-      (assoc :token token)
-      (acl/get-permitting-acls :system-object
-                               e/ingest-management-acl
-                               :update)))
-
-(defn permitted?
-  "Check if a the ACLs for the given token include the given grant and group IDs."
-  [token grant-id group-id]
-  (let [acls (get-system-ingest-update-acls token)]
-    (and (e/grant-permitted? grant-id acls)
-         (e/group-permitted? group-id acls))))
-
-(defn not-permitted?
-  [& args]
-  (not (apply permitted? args)))
+;; XXX Note that once the code in cmr.ingest.api.variables was changed to
+;;     generate an api response using cmr.ingest.api.core/generate-ingest-response
+;;     instead of a custom, private function, the assertion function below
+;;     no longer worked since it couldn't check the key values before they were
+;;     transformed by `generate-ingest-response`. If this isn't important, we
+;;     can delete this function as well as util/assert-convert-kebab-case.
```

Comment:
After chatting about this, @ygliuvt  suggested moving services and variables to XML responses instead of JSON, since this is what the rest of ingest currently uses. This won't fix the bug, but does make the point moot for this PR.